spec/services/arcgis_api/geocoder_spec.rb

```diff
@@ -33,4 +33,29 @@
       )
     end
   end
+
+  describe '#find_address_candidates' do
+    it 'returns candidates from magic_key' do
+      stub_request_candidates_response
+
+      suggestions = subject.find_address_candidates('abc123')
+
+      expect(suggestions.first.address).to be_present
+      expect(suggestions.first.address).to be_present
+      expect(suggestions.first.location).to be_present
+      expect(suggestions.first.street_address).to be_present
+      expect(suggestions.first.city).to be_present
+      expect(suggestions.first.state).to be_present
+      expect(suggestions.first.zip_code).to be_present
```

Comment:
On that note, would it also be wise to test that the suggestion includes _exactly_ the properties we're expecting, so that the test doesn't give us false confidence if we add a new property that isn't tested here currently?

For example:

```suggestion
      expect(suggestions.first.to_h).to eq(
        address: '...',
        # ...
      )
```